.github/workflows/frontend_tests.yml

```diff
@@ -76,6 +76,7 @@ jobs:
         with:
           name: frontend-coverage-${{ matrix.num_runs }}
           path: /home/runner/work/oppia/karma_coverage_reports
+          retention-days: 1
```

Comment:
@U8NWXD Is this needed for more than one day?

Reply:
I'm assuming 1 day here means 24 hours. In that case, it SGTM.

Reply:
If this is not intermediary and you need to do some action I think it is fine to set it to like 7 days

Reply:
Or even keep it as is.

Reply:
Then let's make this 7 days since we use the artifact to diagnose frontend failures.

Reply:
Hmm, I will rather keep it at the default value to have consistency with other artefacts,